src/api/v2/paths/directory.js

```diff
@@ -51,7 +51,9 @@ const get = async (req, res) => {
         return errorBuilder(res, 400, ['Size Limit Exceeded (search too broad)']);
       }
       if (ldeMessage.includes('Time Limit Exceeded')) {
-        return errorBuilder(res, 400, ['Request timed out. You are most likely seeing this if your query is too broad, causing a timeout on the database end. Try limiting query to be more specific']);
+        return errorBuilder(res, 400, ['Request timed out. You are most likely seeing this if '
+        + 'your query is too broad, causing a timeout on the database end. Try limiting query to '
+        + 'be more specific']);
```

Comment:
Try using this style: https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript

Reply:
Nevermind, I was drunk 🍻 Please ignore my comment here.